src/main/xsd/state/1.0.0/state.xsd

```diff
@@ -2705,6 +2705,24 @@
             </xs:annotation>
           </xs:element>
 
+          <xs:element name="showRotation" type="xs:boolean" minOccurs="0"
+            maxOccurs="1">
+            <xs:annotation>
+              <xs:documentation xml:lang="en" xml:space="preserve">
+
```

Comment:
@welchyd Could you provide documentation for the new elements?

Reply:
Very well. Added LOTS of documentation :)